### ISSUE TITLE
The Web Components deployment should be adapted for Web Components refactor

### ISSUE THREAD

**Issue by willmcvay** [reapit/foundations#124]
Depends on #559 
- Web components need to now deploy multiple components, multiple builds for each and a serverside component for each.
- server-side handlers should deploy to AWS Lambda using Serverless config in repo
- Client-side components should export a JS and CSS bundle to a CDN (UMD) and ESM/CJS distributions to NPM.
- Currently there is only one component (search widget), but the scripts should be flexible enough to deploy more in the future.
- All search widgets should deploy to demo-site. Demo site HTML should be refactored to be an ejs template so we can build in the API key at compile / deploy time

**Comment by tanphamhaiduong** [comment]
@willmcvay there is not bucket for webcomponent-dev, so we need to create it

**Comment by tanphamhaiduong** [comment]
@willmcvay , @HollyJoyPhillips this ticket has been descoped only release server side from serverless and release for search-widget to CDN on cloudfront
